### PR TITLE
Lidar docker image

### PR DESCRIPTION
**Pull request by Sithyphus**
Merging lidar image into the test development branch.

### CODE REVIEW

File: lidar-plot-base-transformer/configuration.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+TRANSFORMER_VERSION = '1.0'
+
+# The transformer description
+TRANSFORMER_DESCRIPTION = 'Base for plot-level Lidar-based algorithm transformers'
+
+# Short name of the transformer
+TRANSFORMER_NAME = 'lidar-plot-level-base'
+
+# The sensor associated with the transformer
+TRANSFORMER_SENSOR = ''
```
Comment (Chris-Schnaufer) [inline lidar-plot-base-transformer/configuration.py]: Sensor should be `scanner3DTop`

File: lidar-plot-base-transformer/transformer.py [hunk 2]
```diff
@@ -0,0 +1,882 @@
...
+            proj = osr.SpatialReference(wkt=src.GetProjection())
+
+            return proj.GetAttrValue('AUTHORITY', 1)
+        except Exception as ex:
+            logging.exception("[get_epsg] Exception caught processing file: %s", filename)
+
+        return None
+
+    @staticmethod
+    def get_centroid_latlon(filename: str):
```
Comment (Chris-Schnaufer) [inline lidar-plot-base-transformer/transformer.py]: This won't work for LAS files. See https://github.com/AgPipeline/transformer-plotclip/blob/ca48c9805d555bb42192bab8b682a361a542b5b5/transformer.py#L69

**Review by Chris-Schnaufer — COMMENTED** [review]
The code in this file needs to be changed to use LAS functions instead of image functions.

File: lidar-plot-base-transformer/transformer.py (hunk 2 above)
Comment (Sithyphus) [reply lidar-plot-base-transformer/transformer.py]: Ooh cool thank you, I'll take a look at it.

Comment (Sithyphus) [reply lidar-plot-base-transformer/transformer.py]: In that same file it has a `get_las_epsg_from_header` function where this file has just the `get_espg` function, should be looking at that as well?

Comment (Sithyphus) [reply lidar-plot-base-transformer/transformer.py]: yeah, this function calls `get_las_espg_from_header` so I'll have a lot of restructuring to do.

**Pull request merged**